pkgs/sharkdp/hyperfine/registry.yaml

```diff
@@ -0,0 +1,26 @@
+packages:
+- type: github_release
+  repo_owner: sharkdp
+  repo_name: hyperfine
+  description: A command-line benchmarking tool
+  asset: "hyperfine-{{.Version}}-{{.Arch}}-{{.OS}}.{{.Format}}"
+  format: tar.gz
+  format_overrides:
+    - goos: windows
+      format: zip
```

Comment:
```suggestion
  - goos: windows
    format: zip
```

Reply:
Sorry, I was fighting my formatter. Prettier doesn't have a config option for lack of indentation so I've had to turn it off when working with aqua-registry. yamllint also has a default for indented lists but luckily it has a config.

Reply:
No worry.
I enabled yamllint indentation option.
#3163 #3165